.hadolint.yaml

```diff
@@ -0,0 +1,5 @@
+ignored:
```

Comment:
@chrismwendt why did you choose to ignore this set of rules?

Reply:
- DL3015 "Avoid additional packages by specifying --no-install-recommends." It's my understanding that we're not that concerned about image size, but maybe it's worth it. I don't know how much space it saves.
- DL3009 "Delete the apt-get lists after installing something." Same as DL3015.
- SC2002 "Do not switch to root USER." This seemed like it would have a relatively high chance of breaking an image, compared to the other lint rules.
- SC2102 "Ranges can only match single chars (mentioned due to duplicates)." There's actually no documentation for this rule, and I didn't fully understand what the problem was.

Do you think it's worth fixing these?

Reply:
> we're not that concerned about image size

If it is as easy as adding a flag or a simple `rm` then seems like we might as well do that.

> SC2002 "Do not switch to root USER."

I see `SC2002: Useless cat. Consider cmd < file | .. or cmd file | .. instead.`, which seems easy enough to fix.

> SC2102

Weird there isn't doc. What is an example line in a dockerfile that failed?

In any case, I would add comments to the file to explain why these rules are ignored.

Reply:
I think with stuff like deleting apt-get lists it's not just the image size, it's about not creating a different file system state and accidentally busting the layer cache

Reply:
Ah, I had a typo, I was looking at `3`002 instead of `2`002.

@felixfbecker Good point!